app/Http/Controllers/Admin/WhitelistController.php

```diff
@@ -44,30 +44,27 @@ public function postShow(Request $request) {
 		]);
 
 		foreach ($request->input('responses') as $userId => $response) {
-			switch ($response) {
-				case 'accept':
-					$user = User::find(intval($userId));
-					if ($user === null) {
-						continue;
-					}
-					$user->wants_email = 'yes';
-					$user->save();
-					$user->notify();
+			if ($response == 'accept') {
```

Comment:
Would've probably been fine to keep since breaking out of the switch and continuing the loop would have the same effect in this case. But this is more transparent, and arguably cleaner, so i think it is good.

Reply:
Well I got an error with the previous code so I couldn't keep it as is :p